assets/pickup-availability.js

```diff
@@ -11,7 +11,11 @@ if (!customElements.get('pickup-availability')) {
     }
 
     fetchAvailability(variantId) {
-      const variantSectionUrl = `${this.dataset.baseUrl}variants/${variantId}/?section_id=pickup-availability`;
+      let rootUrl = this.dataset.rootUrl;
+      if (!rootUrl.endsWith("/")) {
+        rootUrl = rootUrl + "/";
+      }
+      const variantSectionUrl = `${rootUrl}variants/${variantId}/?section_id=pickup-availability`;
 
```

Comment:
Good catch

> Let me know if you're ok with adding this fix in this PR as well, otherwise I can create a follow up issue and PR

I think since it is unrelated to the specific of the issue being solved here, I would prefer to have this shipped and iterated over.

I would also appreciate since you are offering to have you take over this specific change. I'm not familiar with the specifics of the JS enhancements in Dawn(in themes in general) to be confident as to the blast radius here.

Reply:
Just for sake of visibility, I've created the follow up PR about this issue here: https://github.com/Shopify/dawn/pull/855

But nothing is needed, I requested reviews from my teammates.